composer.json

```diff
@@ -18,38 +18,38 @@
     ],
     "require": {
         "php": "^5.3 || ^7.0",
-        "symfony/console": "^2.3",
-        "symfony/form": "^2.3",
-        "symfony/framework-bundle": "^2.3",
-        "symfony/templating": "^2.3",
-        "sonata-project/admin-bundle": "^3.4",
-        "sonata-project/user-bundle": "^3.0",
-        "sonata-project/media-bundle": "^3.0",
-        "sonata-project/classification-bundle": "^3.0",
-        "sonata-project/formatter-bundle": "^3.0",
-        "sonata-project/easy-extends-bundle": "^2.2",
-        "sonata-project/intl-bundle": "^2.2.4",
+        "sonata-project/admin-bundle": "^3.23.0",
+        "sonata-project/classification-bundle": "^3.3.1",
+        "sonata-project/core-bundle": "^3.4.0",
         "sonata-project/datagrid-bundle": "^2.2.1",
-        "sonata-project/core-bundle": "^3.0",
-        "swiftmailer/swiftmailer": "^4.3 || ^5.0"
+        "sonata-project/easy-extends-bundle": "^2.2.0",
+        "sonata-project/formatter-bundle": "^3.2.2",
+        "sonata-project/intl-bundle": "^2.3.2",
+        "sonata-project/media-bundle": "^3.6.0",
+        "sonata-project/user-bundle": "^3.2.4",
+        "swiftmailer/swiftmailer": "^4.3 || ^5.0",
+        "symfony/console": "^2.3 || ^3.0",
+        "symfony/form": "^2.3 || ^3.0",
+        "symfony/framework-bundle": "^2.3 || ^3.0",
+        "symfony/templating": "^2.3 || ^3.0"
```

Comment:
Do we need to add Symfony 3 compatibility to fix the build?

Not sure if it will be compatible, but on travis there is no build that checks it, so maybe we should add it on dev-kit, and then create another PR